src/test/resources/network/brightspots/rcv/test_data/2013_minneapolis_mayor/2013_minneapolis_mayor_config.json

```diff
@@ -19,7 +19,7 @@
     "provider" : "ess",
     "treatBlankAsUndeclaredWriteIn": false,
     "overvoteLabel": "overvote",
-    "undervoteLabel": "undervote",
+    "skippedRankLabel": "undervote",
```

Comment:
not a big deal, but ideally we would change all of these test config "undervote" values to "skipped rank" or whatever

Reply:
Agree, downside is that we'd have to update all the CVRs, and those are XLSXs, which makes it hard to just `sed` find/replace each instance of `undervote`